src/table/__integ__/inline-editing.test.ts

```diff
@@ -62,6 +65,42 @@ test(
   })
 );
 
+test(
+  'success icon is displayed and aria live region is rendered in after a successful edit',
+  setupTest(async page => {
+    await page.click(cellRoot$);
+    await page.click(cellSaveButton.toSelector());
+    await expect(page.isFocused(cellEditButton$)).resolves.toBe(true);
+    await expect(page.isDisplayed(successIcon$)).resolves.toBe(true);
```

Comment:
It would make sense to also test that the icon was *not* there before

Reply:
Added the additional checks to validate the elements are not existing before the cell edit process is starting.